tests/atoms/gaia_data_layer.js

```diff
@@ -495,13 +495,15 @@ var GaiaDataLayer = {
     };
   },
 
-  sendSMS: function(recipient, content, aCallback) {
+  sendSMS: function(recipient, content, synchronous, aCallback) {
```

Comment:
We can't call `synchronous` this way. This variables doesn't make the function synchronous, it just by passes the wait for sms to come. Let's rename it to `byPassSendVerification`.